crisischeckin/CrisisCheckinMobile/CrisisCheckinMobile/ViewModels/ProfileViewModel.cs

```diff
@@ -0,0 +1,187 @@
+        {
+            get
+            {
+                return new Command(async () =>
+                {
+                    //TODO 
+                    await navigation.PushAsync(new TemporaryView());
+                });
+            }
+        }
+
+        public ICommand GoToViewDisasterData
+        {
+            get
+            {
+                return new Command(async () =>
+                {
+                    //TODO 
+                    await navigation.PushAsync(new TemporaryView());
+                });
+            }
+        }
+
+        public ICommand GoToViewDisasterResources
+        {
+            get
+            {
+                return new Command(async () =>
+                {
+                    //TODO 
+                    await navigation.PushAsync(new TemporaryView());
+                });
+            }
+        }
+
+        public ICommand AddResources
+        {
+            get
+            {
+                return new Command(async () =>
+                {
+                    //TODO 
+                    await navigation.PushAsync(new TemporaryView());
+                });
+            }
+        }
+
+        public ICommand AddContacts
+        {
+            get
+            {
+                return new Command(async () =>
+                {
+                    //TODO 
+                    await navigation.PushAsync(new TemporaryView());
+                });
+            }
+        }
+
+        public ICommand ReportData
+        {
+            get
+            {
+                return new Command(async () =>
+                {
+                    //TODO
+                    await navigation.PushAsync(new TemporaryView());
+                });
+            }
+        }
+
+        public ICommand CheckOutOfDisaster
+        {
+            get
+            {
+                return new Command(() =>
+                {
+                    this.IsCheckedInVisible = true;
+                    this.IsCheckedOutVisisble = false;
+                });
+            }
+        }
+
+        public ICommand CheckInToDisaster
+        {
+            get
+            {
+                return new Command(() =>
+                {
+                    // TODO call to check in page
+                    //await navigation.PushAsync(new TemporaryView());
+                    this.IsCheckedInVisible = false;
+                    this.IsCheckedOutVisisble = true;
+                });
+            }
+        }
+
+        private bool isCheckedIn = true;
+        public bool IsCheckedInVisible
+        {
+            get
+            {
+                return isCheckedIn;
+            }
+            set
+            {
+                if (isCheckedIn != value)
+                {
+                    isCheckedIn = value;
+
+                    PropertyChanged?.Invoke(this, new PropertyChangedEventArgs(nameof(IsCheckedInVisible)));
+                }
+            }
+        }
+
+        private bool isCheckedout;
```

Comment:
As I read the code, I think I'd prefer seeing one field for checked in (or checked out). And have the properties of the View Model create the two public properties for data binding the visibility. I think it would make the design clearer.